econml/tests/test_automated_ml.py

```diff
@@ -154,28 +152,21 @@ def test_param(self):
 
     def test_forest_dml(self):
         """Testing the completion of the fit and effect estimation of an AutomatedForestDML"""
-
-        Y, T, X, _ = ihdp_surface_B()
-        est = AutomatedForestDML(model_y=automl_model_reg(),
-                                 model_t=GradientBoostingClassifier(),
-                                 discrete_treatment=True,
-                                 n_estimators=1000,
-                                 subsample_fr=.8,
-                                 min_samples_leaf=10,
-                                 min_impurity_decrease=0.001,
-                                 verbose=0, min_weight_fraction_leaf=.01)
+        est = AutomatedCausalForestDML(model_y=automl_model_reg(),
```

Comment:
where did the data generation go?

Reply:
We were already performing the data generation in the big `try` block up top, and we made use of this fact in one test but not the others, which I found confusing.